convert2rhel/main.py

```diff
@@ -165,6 +165,10 @@ def main_locked():
 
     except _AnalyzeExit:
         breadcrumbs.breadcrumbs.finish_collection(success=True)
+        # Update RHSM custom facts only when this returns False. Otherwise,
+        # sub-man get uninstalled and the data is removed from the server.
```

Comment:
```suggestion
        # sub-man get uninstalled and the data is removed from the RHSM server.
```
I would change it a bit to have easier understanding